apps/src/templates/instructions/InstructionsWithWorkspace.jsx

```diff
@@ -22,6 +22,11 @@ var InstructionsWithWorkspace = React.createClass({
     instructionsMaxHeight: React.PropTypes.number.isRequired,
     setInstructionsHeight: React.PropTypes.func.isRequired,
     setInstructionsMaxHeight: React.PropTypes.func.isRequired,
+
+    isCsf: React.PropTypes.bool,
+    // TODO - properly pass these to top instructions
+    isRtl: React.PropTypes.bool.isRequired,
+    noVisualization: React.PropTypes.bool.isRequired
```

Comment:
Yeah. While topInstructions were CSP only, we didn't need to worry about this. In CSF we will going forward. I'm not using this yet, but will need to eventually (hence the TODO)